src/main/resources/default/templates/biz/tycho/analytics/data-explorer.html.pasta

```diff
@@ -14,6 +14,9 @@
                 <button type="button" id="addBtn" class="btn btn-primary mr-2" onclick="openTypeSelector()">
                     <i class="fa fa-plus"></i> @i18n("DataExplorerController.addChart")
                 </button>
+                <button type="button" id="exportBtn" class="btn btn-link mr-2" onclick="exportCharts()">
+                    <i class="fa-solid fa-download"></i> @i18n("DataExplorerController.export")
+                </button>
```

Comment:
**Optional:** Hide via class as default to prevent short flickering of button when opening data explorer without charts?